### PR TITLE
arangodb 3.9.2

### PR DESCRIPTION
**Pull request by KVS85**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/arangodb.rb
Comment (cho-m) [inline Formula/arangodb.rb:7]: Need to remove revision
```suggestion
```

Comment (KVS85) [reply Formula/arangodb.rb]: Done.

**Comment by KVS85** [comment]
@cho-m unfortunately the result is the same:
```
...
Error: PR requires the CI-long-timeout label but it is not set!
Error: If the longer timeout is not required, remove the "long build" label.
Error: Otherwise, add the "CI-long-timeout" label.
Error: No more than 2 PRs at a time may use "CI-long-timeout".
...
```

**Comment by cho-m** [comment]
That just means you need to wait until maintainer can add a https://github.com/Homebrew/homebrew-core/labels/CI-long-timeout when CI queue is available. There can only be 2 PRs with label and they are already occupied.

**Comment by KVS85** [comment]
@cho-m can you please put https://github.com/Homebrew/homebrew-core/labels/CI-long-timeout once again?

**Comment by KVS85** [comment]
@cho-m can you please merge this PR if possible?

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**
